app/models/concerns/user_permissions.rb

```diff
@@ -215,6 +217,14 @@ def can_edit_some_project_groups
       can_edit_assigned_project_groups? || can_edit_project_groups?
     end
 
+    def can_manage_forms_for_role(role)
+      can_manage_forms? && (Hmis::Form::Definition::NON_ADMIN_FORM_ROLES.include?(role) || can_administrate_config?)
```

Comment:
To make this slightly more robust, we can do `role.to_s` to ensure that `user.can_manage_forms_for_role?(:SERVICE)` works the same way as `user.can_manage_forms_for_role?('SERVICE')`

Reply:
Good point thank you